index.html

```diff
@@ -163,6 +163,20 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
             <img class="separa" src="icons/Separator Bottom.png" alt="Separator">
         </div>
-    </section>  
+    </section> 
+    <section class="contactform">
+        <img class="contactshapes" src="icons/contact form shapes-mobile@2x.png" alt="contactimage">
+        <article class="contactme">
+            <h2 class="cme">Contact me</h2>
+            <p class="pme">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </article>
+        <form class="form" action="https://formspree.io/f/mnqydvzj" method="post">
+            <input name="Name" id="name" type="text" placeholder="Name..." maxlength="30" required><br>
+            <input name="Email" id="email" type="email" placeholder="you@example.com" required><br>
+            <textarea name="message" id="textarea" placeholder="Write your message here..." maxlength="500" required></textarea><br>
+        </form>
+        <button class="submitted" type="submit">Get in touch</button>
```

Comment:
- [ ] You did a great job applying the requested changes by the previous reviewer. 💯 However, your form isn't working because your submit button is outside it! 😱 Please make sure that you move the "Get in touch" button inside the `<form>`. 🤓 🔧 

```suggestion
        <form class="form" action="https://formspree.io/f/mnqydvzj" method="post">
            <input name="Name" id="name" type="text" placeholder="Name..." maxlength="30" required><br>
            <input name="Email" id="email" type="email" placeholder="you@example.com" required><br>
            <textarea name="message" id="textarea" placeholder="Write your message here..." maxlength="500" required></textarea><br>
            <button class="submitted" type="submit">Get in touch</button>
        </form>
```